servicea.yaml

```diff
@@ -34,7 +34,7 @@ spec:
         command: [ "watch" ]
         args: [ "echo `date` | ncat serviceb 8080"]
       - name: kong
-        image: mashape/kong-enterprise:mesh
+        image: kong:1.0-centos
```

Comment:
I guess we could maybe just swap this for `latest`? 🤔

Reply:
Yeah. Could just have this as image `kong`.